test/codegen-generated-classes-test/src/test/java/software/amazon/awssdk/services/CodegenServiceClientConfigurationTest.java

```diff
@@ -82,7 +82,9 @@ <T> void externalInternalTransforms_preserves_propertyValues(TestCase<T> testCas
             ProtocolRestJsonServiceClientConfigurationBuilder.builder(clientConfig.toBuilder());
 
         // Assert that we can retrieve the same value
-        assertThat(testCase.getter.apply(anotherBuilder)).isEqualTo(testCase.value);
+        if (testCase.hasDirectMapping) {
```

Comment:
We not longer "mirror" the client override configuration from the client configuration, therefore this test no longer works for those properties.